account_statement_base/views/account_bank_statement_line.xml

```diff
@@ -112,12 +113,19 @@
                     icon="fa-undo"
                     attrs="{'invisible': [('is_reconciled', '=', False)]}"
                 />
+            <button
+                    name="action_open_journal_entry"
+                    type="object"
+                    string="Open Journal Entry"
```

Comment:
I think there shouldn't be string. And please put title for proper screen readers (for blind people) usage.